.gitignore

```diff
@@ -6,3 +6,4 @@
 .env*
 .vscode
 node_modules
+/vendor
```

Comment:
There should be a newline at end of file.

Reply:
Added